sherpa.sh

```diff
@@ -0,0 +1,40 @@
+package: SHERPA
+version: "%(tag_basename)s"
+source: https://github.com/alisw/sherpa
+tag: "alice/v2.1.1"
+build_requires:
+  - autotools
+  - GCC-Toolchain
+---
+#!/bin/bash -e
+
+rsync -a --delete --exclude '**/.git' --delete-excluded $SOURCEDIR/ ./
+
+autoreconf -ivf
+./configure --prefix=$INSTALLROOT \
+            --with-sqlite3=install
```

Comment:
Ok, what we do here is we build the required sqlite stuff within SHERPA. So also here we avoid the problem of incompatibilities, no?

Sure, a separate package is more flexible and re-usable also for other things. Who takes care of this?